src/sbt-test/scoverage/bad-coverage-file-stmt/build.sbt

```diff
@@ -0,0 +1,14 @@
+version := "0.1"
+
+scalaVersion := "2.10.4"
```

Comment:
@kitbellew the test failures are actually due to these build files that have a 2.10 dependency. scoverage is no longer published for 2.10 or 2.11, so you'll need to bump this up. For now stick in the 2.12 range. I need to update all of the test dependencies here since org.specs2 changed names and is not available for 2.13 here either.

Reply:
> Also my apologies, but I caused a bunch of conflicts. The specs2 that was in here was really old and I wanted to have a newer framework in there to not block any updates or version or anything. So I went through and updated quite a bit of stuff. If you are able to rebase again, and follow the pattern of the changes, it should be good to go.

no problem. rebased, switched to munit and scala 2.13. ptal.